include/MVC/SugarApplication.php

```diff
@@ -704,49 +706,77 @@ public static function getSuccessMessages() {
         return $messages;
     }
 
+
     /**
-     * Storing messages into session
-     * @param string $message
-     */
-    protected static function appendMessage($type, $message) {
-        
-        self::validateMessageType($type);
+     * Set a message which reflects the status of the performed operation.
+     *
+     * @param string $message the message to be displayed to the user.
+     * For consistency with other messages, it should begin with a capital letter and end with a period.
+     *
+     * @param string $type the type of the message. Values allowed: error, info, alert, okay, warning.
+     *
+     * @param boolean $repeat if this is FALSE and the message is already set, then the message won't be repeated.
+     *
+    */
+    public static function appendMessage($message, $type = 'info', $repeat = true)
```

Comment:
Do not change tha places of parameters.

Reply:
Fixed in commit: https://github.com/salesagility/SuiteCRM/pull/6602/commits/163d288e7218bb236cef64775f5d95d146396826